interface/ceed.c

```diff
@@ -365,10 +405,50 @@ int CeedSetDelegate(Ceed ceed, Ceed *delegate) {
   return 0;
 }
 
+/**
+  @brief Set a backend function
+
+  @param ceed           Ceed for error handling
+  @param type           Type of Ceed object to set function for
+  @param[out] object    Ceed object to set function for
+  @param fname          Name of function to set
+  @param f              Function to set
+
+  @return An error code: 0 - success, otherwise - failure
+
+  @ref Advanced
+**/
+int CeedSetBackendFunction(Ceed ceed,
+                           const char *type, void *object,
+                           const char *fname, int (*f)()) {
+  char lookupname[100];
+  strcpy(lookupname, "");
+
+  // Build lookup name
+  if (!strcmp(fname, "Apply") || !strcmp(fname, "Destroy")) {
+    strcat(strcat(lookupname, type), fname);
+  } else {
+    strcat(lookupname, fname);
+  }
+
+  // Find and use offset
+  for (CeedInt i = 0; i < 25; i++) {
+    if (!strcmp(ceed->foffsets[i].fname, lookupname)) {
+      size_t offset = ceed->foffsets[i].offset;
+      size_t *fpointer;
+      fpointer = (size_t *)(object + offset);
+      *fpointer = (size_t) f;
```

Comment:
We don't have checking that the function has the desired inputs, but we never did at this point (setting the pointer).